example/infra/transforms/lint.py

```diff
@@ -3,17 +3,19 @@
 from typing import Iterable, List, Union
 
 from blockwork.build import Transform
+from blockwork.context import Context
 from blockwork.tools import Invocation, Version
 
 from infra.tools.simulators import Verilator
 
 @Transform.register()
 @Transform.tool(Verilator)
 @Transform.input(".sv")
-def verilator_lint(tools    : SimpleNamespace,
+def verilator_lint(ctx      : Context,
+                   tools    : SimpleNamespace,
                    inputs   : SimpleNamespace,
                    out_dirx : Path) -> Iterable[Union[Invocation, Path]]:
-    yield tools.verilator.get_action("run")(
+    yield tools.verilator.get_action(ctx, "run")(
```

Comment:
This feels a bit weird that the action definition contains context, but then it's passed through `get_action` rather in the call to the action - could we make it consistent and pass it through the call of the action instead?